.github/workflows/ci_windows.yaml

```diff
@@ -77,7 +77,7 @@ jobs:
       - name: Install Qt
         uses: jurplel/install-qt-action@v2
         with:
-          version: "5.15.2"
+          version: "6.2.0"
```

Comment:
Is 6.2.0 a latest available release from 6.2 branch?

Reply:
Obviously no, but I choose this version to be in line with other OS CI.
Also I'm not sure what is the maximum version number that `jurplel/install-qt-action` will support.

Reply:
- `jurplel/install-qt-action@v2` still uses `aqtinstall 1.2.5` by default. (No `Qt 6.2.1+` support)
- This will be changed to `aqtinstall 2.x` in `jurplel/install-qt-action@v3`
- `Qt 6.2.1+` support wasn't added to [`aqtinstall until 2.0.2`](https://github.com/miurahr/aqtinstall/releases/tag/v2.0.2)

`jurplel/install-qt-action` maintainer hasn't got around to make an official `v3` release just yet but has a `v3` branch available & it can be used `jurplel/install-qt-action@v3`

>https://github.com/jurplel/install-qt-action/issues/139#issuecomment-1039194328

@Chocobo1 would you be ok with using `jurplel/install-qt-action@v3` now?

`Qt 6.3.0` should be GA in `mid-April` (this will include a fix for #15941, have no ETA for `Qt 6.2.5` which supposedly will also contain this fix.)

`Qt 5.15.3` was also released to `open source community` however it's [not available yet](https://download.qt.io/online/qtsdkrepository/windows_x86/desktop/) to use with `jurplel/install-qt-action`

Reply:
**Suggestion:**
(to be more consistent with other OS CI)

```
    strategy:
      fail-fast: false
      matrix:
        libt_version: ["2.0.5", "1.2.15"]
        qt_version: ["6.2.0"]
```
____

Note - Qt action also bumped here.
```
      - name: Install Qt
        uses: jurplel/install-qt-action@v3
        with:
          version: ${{ matrix.qt_version }}
```
____

```
      - name: Upload build artifacts
        uses: actions/upload-artifact@v3
        with:
          name: qBittorrent-CI_Windows-x64_libtorrent-${{ matrix.libt_version }}_Qt-${{ matrix.qt_version }}
```

Reply:
>https://github.com/jurplel/install-qt-action/issues/139#issuecomment-1039194328@Chocobo1 would you be ok with using jurplel/install-qt-action@v3 now?

I don't think so as it wasn't officially released yet.

>Suggestion:
(to be more consistent with other OS CI)

I don't have strong feelings about it. The Windows CI is too slow so we aren't adding Qt variations. And the artifact naming is for differentiating libraries, so far there it isn't really needed. I mean normally, users shouldn't concern which library is in use because artifact is provided as is (no selection nor customization).